src/process/Convert.js

```diff
@@ -10,17 +10,19 @@ class Convert {
   }
 
   async convert(authorJson) {
-    const output = new Questionnaire(authorJson);
+    const convertedQuestionnaire = new Questionnaire(authorJson);
+
+    const result = await this.schemaValidator.validate(convertedQuestionnaire);
 
-    const result = await this.schemaValidator.validate(output);
     if (!result.valid) {
       throw new ValidationError(
         "Converted author schema is not valid EQ schema.",
-        result.errors
+        result.errors,
+        convertedQuestionnaire
```

Comment:
Think these are being passed to the error class the wrong way round, error class is expecting `constructor(message, input, result)` and this seems to be passing `message, result, input`.